src/ResourceManager/Network/Commands.Network/ApplicationGateway/BackendHttpSettings/AzureApplicationGatewayBackendHttpSettingsBase.cs

```diff
@@ -96,6 +97,7 @@ public class AzureApplicationGatewayBackendHttpSettingsBase : NetworkBaseCmdlet
         [Parameter(
             Mandatory = false,
             HelpMessage = "Flag if probe should be enabled.")]
+        [Obsolete("This parameter is ignored and will be removed in the future.")]
```

Comment:
@jobatzil please change this message to something along the lines of "Parameter ProbeEnabled is ignored and will be removed in a future release."

This will make it easier for users to identify which parameter is being removed.